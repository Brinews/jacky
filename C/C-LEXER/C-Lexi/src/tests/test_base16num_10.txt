
  34Ax16567x16x16
