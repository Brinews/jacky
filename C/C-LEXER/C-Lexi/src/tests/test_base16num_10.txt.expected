{34Ax16 , BASE16NUM , 2}
{567x16 , BASE16NUM , 2}
{x16 , ID , 2}
{ , END_OF_FILE , 4}
